Src/BootCamp.Chapter/Checks.cs

```diff
@@ -14,28 +16,32 @@ namespace BootCamp.Chapter
     /// </summary>
     public static class Checks
     {
+        
         public static int PromptInt(string message)
         {
+            
             // To do: call your implementation. 
-            return 0;
+            return Convert.ToInt32(PromptString(message)); ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4862789)